src/CTA.WebForms/Services/TaskManagerService.cs

```diff
@@ -17,24 +18,34 @@ public class TaskManagerService
         private const string UnstalledLogAction = "Exited Stall";
 
         public enum TaskState { Active, Waiting }
-        public enum TaskStallTimeout { None = 0, Short = 100, Medium = 1000, Long = 10000 }
+        // Predefined timeout periods (in milliseconds) for when the application enters a stall state
+        public enum TaskStallTimeoutMs { None = 0, Short = 100, Medium = 1000, Long = 10000 }
```

Comment:
how did you come up with these numbers?

Reply:
Just used my judgement on what I thought would be a minimum "safe" timeout before cancelling tasks and used that as the value for small, then increased by a factor of 10 for medium and large